docs/docbook5/en/source/chapters/extending.xml

```diff
@@ -376,6 +380,10 @@ class MyEchoTask extends Task {
         $this->message = $str;
     }
 
+    public function setReverse($str) {
+        $this->reverse = StringHelper::booleanValue($str);
```

Comment:
It is not necessary to use `StringHelper::booleanValue()` here as it happens in the
[`IntrospectionHelper`](https://github.com/phingofficial/phing/blob/ac3cc900a34fd0a2647850f367268687135b030b/classes/phing/IntrospectionHelper.php#L326) as well.